README.md

```diff
@@ -1,6 +1,198 @@
-# curriculum-cr-training-mobile-menu
+<a name="readme-top"></a>
 
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+Modify this file to match your project and remove sections that don't apply.
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 CRT Mobile Menu ](#-crt-mobile-menu-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 CRT Mobile Menu <a name="about-project"></a>
+
+**CRT Mobile Menu** is a portfolio website that allows users to view my projects.
+
+
```

Comment:
### [OPTIONAL] :wink: 
- [ ] It looks like this content aimed at telling users the "what" and "why" of your project is too short. README's are important parts of the code too, and it can be used an opportunity to impress visitors, and to show them your commitment to writing detailed and resourceful (professional) documentation or technical writing. I think it will be an added advantage to have an excelling README by telling a nicely written story of what your project is, why you embarked on it, etc. It's almost an opportunity to pitch yourself :smiley: